.github/workflows/ci.yaml

```diff
@@ -58,23 +76,13 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - name: Install NPM packages
-      run: |
-        cd functions/ts
-        npm ci
-    - name: Build node docker images
-      if: matrix.platform == 'ubuntu-latest'
-      run: |
-        cd functions/ts
-        npm run kpt:docker-build
-    - name: Build go images
+    - name: Build node and Go docker images
       if: matrix.platform == 'ubuntu-latest'
       run: |
-        cd functions/go
-        BUILDONLY=true TAG=dev ./publish-functions.sh
-    - name: Run all e2e tests
+        make build-dev
+    - name: Run all tests
       if: matrix.platform == 'ubuntu-latest'
       run: |
         sudo curl https://storage.googleapis.com/kpt-dev/latest/linux_amd64/kpt -o /usr/local/bin/kpt
         sudo chmod +x /usr/local/bin/kpt
-        tests/e2e.sh
+        make TAG=dev e2e-test
```

Comment:
if we have a `make build-dev` target, this this should probably be a `make e2e-test-dev`.